packages/wallet/test/PersonalWallet/methods.test.ts

```diff
@@ -462,6 +464,20 @@ describe('PersonalWallet methods', () => {
         expect(await txPending).toEqual(outgoingTx);
       });
 
+      it('attempts to resubmit the tx that is already in flight', async () => {
```

Comment:
```suggestion
      it('only tracks the first submission as in-flight when there are multiple submission of the same tx', async () => {
```
Describe the expectation

Reply:
Main assertion in this test is `expect(txSubmitProvider.submitTx).toBeCalledTimes(2);` though :thinking:

Reply:
Right I see. I'll retract my suggestion